### PR TITLE
changing name and bundle name to be release ready

### PR DESCRIPTION
**Pull request by cooktheryan**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 #
 # For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
 # redhat.com/operator-bundle:$VERSION and redhat.com/operator-catalog:$VERSION.
-IMAGE_TAG_BASE ?= registry.redhat.io/rhtas-tech-preview/sigstore-rhel9-operator
+IMAGE_TAG_BASE ?= quay.io/rcook/sigstore-rhel9-operator
```
Comment (cooktheryan) [inline Makefile:32]: RESET

File: PROJECT [hunk 2]
```diff
@@ -6,10 +6,10 @@ domain: redhat.com
 layout:
 - go.kubebuilder.io/v3
 plugins:
-  manifests.sdk.operatorframework.io/v2: {}
-  scorecard.sdk.operatorframework.io/v2: {}
-projectName: operator
-repo: github.com/securesign/operator
+  manifests.sdk.secure-sign-operatorframework.io/v2: {}
+  scorecard.sdk.secure-sign-operatorframework.io/v2: {}
```
Comment (Gregory-Pereira) [inline PROJECT:10]: why are we tagging the operator sdk related pieces with securesign?

File: bundle.Dockerfile [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
-COPY bundle/tests/scorecard /tests/scorecard/
+FROM registry.redhat.io/openshift4/ose-operator-registry:v4.14 
 
+# Core bundle labels.
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=rhtas-operator
-LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.channels.v1=alpha
```
Comment (Gregory-Pereira) [inline bundle.Dockerfile:8]: so are we going with:
```
channel: alpha
package: rhtas-operator
```
?

File: config/manager/kustomization.yaml [hunk 4]
```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: registry.redhat.io/rhtas-tech-preview/sigstore-rhel9-operator
-  newTag: sha256:e4a077015f43fb343d8b13739f08d90e9e9229dc0e3a90e42f72a33dbacd979e
+  newName: registry.redhat.io/rhtas-tech-preview/sigstore-rhel9-operator-controller
+  newTag: latest
```
Comment (cooktheryan) [inline config/manager/kustomization.yaml:8]: will need to reset based on new image generated

File: bundle/metadata/annotations.yaml [hunk 5]
```diff
@@ -12,14 +12,3 @@ annotations:
...
-  features.operators.openshift.io/disconnected: "false"
-  features.operators.openshift.io/fips-compliant: "false"
-  features.operators.openshift.io/proxy-aware: "false"
-  features.operators.openshift.io/cnf: "false"
-  features.operators.openshift.io/cni: "false"
-  features.operators.openshift.io/csi: "false"
-  features.operators.openshift.io/tls-profiles: "false"
-  features.operators.openshift.io/token-auth-aws: "false"
-  features.operators.openshift.io/token-auth-azure: "false"
-  features.operators.openshift.io/token-auth-gcp: "false"
```
Comment (Gregory-Pereira) [inline bundle/metadata/annotations.yaml:25]: why are we removing these pieces, this showed up in the official docs for the build process?

**Review by Gregory-Pereira — COMMENTED** [review]
I can see the direction your going with this and it certainly could resolve the issues, I just had some quesitons

File: bundle.Dockerfile (hunk 3 above)
Comment (cooktheryan) [reply bundle.Dockerfile:8]: ya. I say we keep alpha until GA

File: bundle/metadata/annotations.yaml (hunk 5 above)
Comment (cooktheryan) [reply bundle/metadata/annotations.yaml:25]: It was lost due to bundle regeneration

**Review by Gregory-Pereira — APPROVED** [review]
/approve
/lgtm

**Pull request merged**